src/lib/UtapiClient.js

```diff
@@ -169,33 +167,26 @@ export default class UtapiClient {
      */
     _getParamsArr(params) {
         this._checkMetricTypes(params);
+        const props = [];
+        const { byteLength, newByteLength, oldByteLength, numberOfObjects } =
+            params;
+        const metricData = {
+            byteLength,
+            newByteLength,
+            oldByteLength,
+            numberOfObjects,
+        };
         // Only push metric levels defined in the config, otherwise push any
         // levels that are passed in the object
-        const levels = this.metrics ? this.metrics : Object.keys(metricObj);
-        const props = [];
-        for (let i = 0; i < levels.length; i++) {
-            const prop = metricObj[levels[i]];
-            if (params[prop] !== undefined) {
-                props.push(metricObj[levels[i]]);
-            }
+        const levels = this.metrics ? this.metrics : metricLevels;
+        if (params.bucket && levels.indexOf('buckets') >= 0) {
```

Comment:
That's weird? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes do you mean because `buckets` is an array?